client/resolvers.go

```diff
@@ -54,3 +58,32 @@ func ResolveTagField(fieldName string) func(context.Context, schema.ClientMeta,
 		return diag.WrapError(r.Set(c.Name, data))
 	}
 }
+
+func parseISODate(d string) (*time.Time, error) {
+	if d == "" {
+		return nil, nil
+	}
+	location, err := time.LoadLocation("UTC")
+	if err != nil {
+		return nil, err
+	}
+	date, err := time.ParseInLocation(time.RFC3339, d, location)
+	if err != nil {
+		return nil, err
+	}
+	return &date, err
+}
+
+func ISODateResolver(path string) schema.ColumnResolver {
```

Comment:
could use `schema.DateResolver` instead?

Reply:
Sure. I didn't know that it does exist in core.